src/shapes/text.class.js

```diff
@@ -169,6 +160,13 @@
     lineHeight:           1.16,
 
     /**
+     * deltaY
+     * @type Number
+     * @default
+     */
+    deltaY:               0,
```

Comment:
i m uncertain about this. We should not offer a global deltaY property to the developer.

There is a probably a way to manage style without having the parent property set.